gtd-timer/Controllers/PresetController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+using ServiceTier.Services;
+using Common.ModelsDTO;
+using Microsoft.AspNetCore.Authorization;
+
+namespace gtdtimer.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    public class PresetController : ControllerBase
+    {
+        private readonly IUserIdentityService userIdentityService;
+        private readonly IPresetService presetService;
+
+        public PresetController(IUserIdentityService userIdentityService, IPresetService presetService)
+        {
+            this.userIdentityService = userIdentityService;
+            this.presetService = presetService;
+        }
+
+        [HttpGet("[action]")]
+        public IActionResult GetPreset(int presetid)
+        {
+            var preset = presetService.GetPresetById(presetid);
+
+            return Ok(preset);
+        }
+
+        [HttpGet("[action]")]
+        public IActionResult GetAllCustomPresets()
```

Comment:
name is a bit misleading - we are not actually getting *all* presets (I hope), just for specific user.